searx/network/client.py

```diff
@@ -136,7 +135,7 @@ def get_transport_for_socks_proxy(verify, http2, local_address, proxy_url, limit
     return AsyncProxyTransportFixed(proxy_type=proxy_type, proxy_host=proxy_host, proxy_port=proxy_port,
                                     username=proxy_username, password=proxy_password,
                                     rdns=rdns,
-                                    loop=LOOP,
+                                    loop=get_loop(),
```

Comment:
I rebased and pushed force ... BTW I added this replacement for the global LOOP variable .. I guess you missed it.

Reply:
Can you add another commit rather than push force, so I can easy see what has been changed ?

Reply:
> Can you add another commit

sure .. sorry.

Reply:
NP